hadoop-ozone/ozonefs-hadoop2/pom.xml

```diff
@@ -54,6 +54,16 @@
       <artifactId>hadoop-common</artifactId>
       <scope>provided</scope>
       <version>2.7.3</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-auth</artifactId>
```

Comment:
The version for `hadoop-auth` and `hadoop annotations` might be different here as this package is explicitly built to take `dependency` on `hadoop 2.7.3` vs `hadoop3` based packages. We might have to suppress any errors that might arise due to this package depending on a different version of hadoop.

Reply:
To explicitly provide these jars to hadoop-2, explicitly copying them in `hadoop-dist/pom.xml`

Reply:
I am not sure I follow why `hadoop-ozone/dist/pom.xml` and this file need to change. When building ozonefs for Hadoop2 we explicitly want to take hadoop2 as a dependency for everything.
Also, do you plan to add the enforcement for `DependencyConvergence`

Reply:
I see the problem, we will need to figure out how to keep all hadoop dependencies in `ozonefs-hadoop2` to be with version 2 and the rest of the packages use hadoop3 as their dependency.
@adoroszlai any recommendation?

Reply:
`hadoop-auth` and `hadoop-common` are present in ozonfs-hadoop2 classpath with the version 2.7.3. This was verified by looking at the classpath generated by ```mvn dependency:build-classpath ```. And this jar is being explicitly made available by the copy command in `dist/pom.xml`. For all other modules, we are defaulting to the 3.3.4 version. 
This is exactly what we intended to. Why do you think we still have a problem?